components/script/timers.rs

```diff
@@ -418,8 +418,8 @@ enum InternalTimerCallback {
     ),
 }
 
-impl JsTimers {
-    pub fn default() -> JsTimers {
+impl Default for JsTimers {
```

Comment:
You'll need to have the `default()` method inside a block like this:

```rust
impl Default for JsTimers {
 ...
}
```

but the rest of the methods will need to be inside a block that looks like this:

```rust
impl JsTimers {
 ...
}
```